pkg/db/dto/project.go

```diff
@@ -10,16 +10,24 @@ import (
 	"github.com/daytonaio/daytona/pkg/workspace/project/config"
 )
 
+type CloneTarget string
+
+const (
+	CloneTargetBranch CloneTarget = "branch"
```

Comment:
`CloneTargetBranch` and `CloneTargetCommit`
Why is this defined here and where is it used?

Why isn't gitprovider.CloneTarget used?